pkg/apis/keycloak/v1alpha1/keycloak_types.go

```diff
@@ -45,7 +45,7 @@ var (
 	NoPhase          StatusPhase
```

Comment:
The version is not updated in the status. Need to find a way to get this dynamically. Any suggestions?

Reply:
What about specifying a tag in addition to the image [here](https://github.com/keycloak/keycloak-operator/blob/master/pkg/model/constants.go#L13)? Then we could use the tag as the version.

Reply:
Yes, this is exactly what I have suggested on Sebastians PR https://github.com/keycloak/keycloak-operator/pull/43#discussion_r332582644